### ISSUE TITLE
Feature request: Template variable support

### ISSUE THREAD

**Issue by kthix** [terraform-providers/terraform-provider-panos#163]
Hi, 

I would like to be able to configure a variable value for a certain device. This can automate interface configuration by setting the value in Panorama for the variable.

Regards,

Kevin

**Comment by shinmog** [comment]
@kthix 
Both Panorama template and template stack variables are there now:

    https://www.terraform.io/docs/providers/panos/r/panorama_template_variable.html

Or is it something else you're referring to?

**Comment by kthix** [comment]
@shinmog 
Yes that is already great. But then you need to fill in the value of those variables. That is not possible. You need to tell "serial device" $fw-ip value ip-subnet 10.0.0.1
That is in the webui done under Panorama=> managed devices=>summary then find the serial number and click edit in the variable column.

**Comment by jochristian** [comment]
Any chance for supporting this? No updates it seems. 
Is this project dead?

**Comment by ChrisJohnsonBMGF** [comment]
Checking in on this, we'd love to have this.  Is this going to happen?  We'd love to be able to set the template stack variable override.  What's the timeline on this being solved?  Are there other work arounds for this?
